### DISCUSSION TITLE
Styling based on parent state for child within child

### DISCUSSION THREAD

**Discussion by risangbaskoro** [tailwindlabs/tailwindcss#7541] [Help] [answered]
I have this layout:
```html
<div class="group ...">
    <div class="group-hover:bg-green-500">
        ...
    </div>

    <div class="group ...">
        <img class="group-hover:scale-110 ..." src="...">
        <span class="..."></span>
        ...
    </div>
</div>
```

How can I prevent the `img` component to be 110% scaled but still make the first child `div` background changed whenever I hover over the root `div`?

**Comment by MichaelAllenWarner** [comment, marked as answer]
You can't, because the `group-` modifiers use the [descendant combinator](https://developer.mozilla.org/en-US/docs/Web/CSS/Descendant_combinator). They're a little dangerous in this way, IMO, especially because Tailwind is generally used in heavily componentized contexts—i.e., this "nested `group-` problem" can involve multiple files, making it difficult to spot in development.

For this reason, I include a custom `parent-` modifier in my "stock" Tailwind setup. It uses the [child combinator](https://developer.mozilla.org/en-US/docs/Web/CSS/Child_combinator), so it _only_ works for direct parent-child relationships, but if I _can_ use it instead of `group-` then I do. Here's a little demo: https://play.tailwindcss.com/tWrJX5Rapi

**Reply by risangbaskoro** [reply]
It works! Thank you so much
